### ISSUE TITLE
problema na quest classe 1 de mercador

### ISSUE THREAD

**Issue by Nipodemos** [eventMacrosBR/up_1_ao_99_todas_classes-bro#86]
http://openkorebrasil.org/index.php?/topic/3699-020218up-1-99-arruaceirotemplárioalquimistasacerdotecavaleiro-por-nipodemos-e-vtr002/?p=34948

**Comment by vitorsilverio** [comment]
parece que mercador T. também está. Acho que a LuG mudou coisas. Assim como tinha renomeado o Bispo lá

**Comment by Nipodemos** [comment]
verdade, é a única explicação

**Comment by Nipodemos** [comment]
olhei lá, e a coordenada do npc está correta...
ele pode ter tido um problema isolado, ou a sequencia de conversa de quando vc é mercador Trans é diferente

**Comment by vitorsilverio** [comment]
tem um mercador para rebornar?

**Comment by Nipodemos** [comment]
nops hahahaa, longe disso

**Comment by vitorsilverio** [comment]
Este bug ainda acontece?

**Comment by aDwCarrazzone** [comment]
Oi, não sei se já arrumaram, eu tenho um Alquimista pra rebornar. Precisam que faça algum teste?

**Comment by Nipodemos** [comment]
@aDwCarrazzone na verdade sim, ainda precisamos testar, se vc puder fazer isso por nós seria ótimo :smile: 

até hoje não tive a coragem de ir lá e upar um char até 99 pra testar kkkkk

**Comment by aDwCarrazzone** [comment]
Certo, vou rebornar ele e testar. Só eu juntar um zeny pq to pobre.
Aproveitando, alguém sabe uma build pra ele? Criei por divertimento e nunca joguei com ele.

**Comment by Nipodemos** [comment]
oh mano, faz build de MVP
é baseado no AD, fica caro mas é bem divertido, porque vc é poderoso heheh

```
Level 99:

For: 60
Int: 80
Vit: 70
Dex 60

Level 175:

FOR: 90~100
AGI: 1
INT: 110~120 (para facilitar o Instant Cast por atributos)
VIT: 96
DEX: 120
SOR: 1 ou em múltiplo de três
```

aqui ensina sobre habilidades e o resto: [Guia Bioquímico MVP](http://sites.levelupgames.com.br/forum/ragnarok/showthread.php?163140-Guia-Bioqu%EDmico-MVP-by-Roxas-al-Thor)
